react/src/__tests__/pages/AntMedia.test.js

```diff
@@ -1841,7 +1841,7 @@ describe('AntMedia Component', () => {
     });
 
     await waitFor(() => {
-      expect(webRTCAdaptorPlaySpeedTestConstructor).not.toBe(undefined);
+      expect(webRTCAdaptorPlaySpeedTestConstructor).toBe(undefined);
```

Comment:
remove unused `webRTCAdaptorPlaySpeedTestConstructor`

Reply:
it's removed